infrastructure/protocol-upgrade/src/hyperchain-upgrade.ts

```diff
@@ -128,54 +165,54 @@ async function deploySharedBridgeL2Implementation() {
-    await spawn('zk env zksync_local');
-    env.reload('zksync_local');
-    env.modify(
-        'CONTRACTS_ERA_DIAMOND_PROXY_ADDR',
-        process.env.CONTRACTS_DIAMOND_PROXY_ADDR,
-        `etc/env/l1-inits/${process.env.L1_ENV_NAME ? process.env.L1_ENV_NAME : '.init'}.env`
-    );
-    env.modify(
-        'CONTRACTS_L2_SHARED_BRIDGE_ADDR',
-        process.env.CONTRACTS_L2_ERC20_BRIDGE_ADDR,
-        `etc/env/l2-inits/${process.env.ZKSYNC_ENV}.init.env`
-    );
-    env.modify(
-        'CONTRACTS_BASE_TOKEN_ADDR',
-        '0x0000000000000000000000000000000000000001',
-        `etc/env/l2-inits/${process.env.ZKSYNC_ENV}.init.env`
-    );
-
-    await deploySharedBridgeL2Implementation();
-
-    process.chdir(`${process.env.ZKSYNC_HOME}`);
-    await spawn('zk config compile zksync_local');
-    env.reload('zksync_local');
-    // process.chdir(`${process.env.ZKSYNC_HOME}/infrastructure/protocol-upgrade`);
-    await hyperchainUpgrade1();
-    env.reload('zksync_local');
-
-    await insertAddresses();
-    env.reload('zksync_local');
-
-    await spawn('zk f yarn  workspace protocol-upgrade-tool start facets generate-facet-cuts');
-    await spawn('zk f yarn  workspace protocol-upgrade-tool start system-contracts publish-all');
-    await spawn(
-        'zk f yarn  workspace protocol-upgrade-tool start l2-transaction complex-upgrader-calldata --use-forced-deployments --use-contract-deployer'
-    );
-    await spawn('zk f yarn  workspace protocol-upgrade-tool start crypto save-verification-params');
-    await preparePostUpgradeCalldata();
-    await spawn(
-        `zk f yarn  workspace protocol-upgrade-tool start transactions build-default --upgrade-timestamp 1711451944 --zksync-address ${process.env.CONTRACTS_DIAMOND_PROXY_ADDR} --use-new-governance --upgrade-address ${process.env.CONTRACTS_HYPERCHAIN_UPGRADE_ADDR} --post-upgrade-calldata`
-    );
+    // process.chdir(`${process.env.ZKSYNC_HOME}`);
+
+    // await spawn(
+    //     'cp etc/env/.init.env etc/env/l1-inits/.init.env && rm ./etc/env/l2-inits/zksync_local.init.env && rm ./etc/env/target/zksync_local.env'
+    // );
+    // await spawn('zk env zksync_local');
+    // env.reload('zksync_local');
+    // env.modify(
+    //     'CONTRACTS_ERA_DIAMOND_PROXY_ADDR',
+    //     process.env.CONTRACTS_DIAMOND_PROXY_ADDR,
+    //     `etc/env/l1-inits/${process.env.L1_ENV_NAME ? process.env.L1_ENV_NAME : '.init'}.env`
+    // );
+    // env.modify(
+    //     'CONTRACTS_L2_SHARED_BRIDGE_ADDR',
+    //     process.env.CONTRACTS_L2_ERC20_BRIDGE_ADDR,
+    //     `etc/env/l2-inits/${process.env.ZKSYNC_ENV}.init.env`
+    // );
+    // env.modify(
+    //     'CONTRACTS_BASE_TOKEN_ADDR',
+    //     '0x0000000000000000000000000000000000000001',
+    //     `etc/env/l2-inits/${process.env.ZKSYNC_ENV}.init.env`
+    // );
+
+    // await deploySharedBridgeL2Implementation();
+
+    // process.chdir(`${process.env.ZKSYNC_HOME}`);
+    // await spawn('zk config compile zksync_local');
```

Comment:
please remove/uncomment this code. alternatively provide a link to the linear task to resolve it